rel/commands/run

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+release_ctl eval --mfa "Mix.Tasks.Mana.run/1" --argv -- --extra "$@"
```

Comment:
I think we really need to start testing the release so that we can avoid implementation discrepancies...

Reply:
Yeah. We could even do a very simple test after build like `mana self-diagnose` that runs some general tests and makes sure everything looks in line. That would be useful for users of mana, as well.